src/asciidoc/docs/develop/useGit.adoc

```diff
@@ -93,10 +95,25 @@ You will need to use this key each time you clone, pull, or push to the origin r
 WARNING: After you close the window, the access key will be displayed only by name (for example, `Token 1`).
 You will not be able to retrieve it from the Koji site again.
 You can, however, revoke your token and generate a new one if you have forgotten it, or if you believe that someone unauthorized has discovered it.
++
+NOTE: Keep the dialog open for the next two steps.
```

Comment:
Remove this note. It seems more distracting than helpful, especially since you have clear instructions for how to reopen the dialog box where relevant.